docs/index.rst

```diff
@@ -180,6 +180,9 @@ and :func:`itertools.zip_longest`.
 
 exceptions
 ----------
+(These are only supported for older versions of Trio that used ``MultiError``
+rather than ``ExceptionGroup``.)
```

Comment:
(I'm feeling sad, but thanks. 💪  )

Reply:
there may be a need for these utilities even with ExceptionGroup, but defer the work until someone needs it